### PR TITLE
[ACS-3863] Edit aspect dialog accessibility issues

### PR DESCRIPTION
**Pull request by rbahirsheth**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
There are some accessibility issues
1. Form elements must have labels
2. Banner landmark should not be contained in another landmark
3. Page should contain a level-one heading

**What is the new behaviour?**
Fix above issues.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/content-services/src/lib/aspect-list/aspect-list.component.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
                     <mat-checkbox class="adf-aspect-list-check-button" [id]="'aspect-list-'+colIndex+'-check'"
                                   [checked]="nodeAspects?.includes(aspect?.entry?.id)"
                                   (click)="onCheckBoxClick($event)"
-                                  (change)="onChange($event, aspect?.entry?.id)">
+                                  (change)="onChange($event, aspect?.entry?.id)"
+                                  [aria-label]="'aspect-list-'+colIndex+'-check'">
```
Comment (DenysVuika) [inline lib/content-services/src/lib/aspect-list/aspect-list.component.html:10]: I don't think this label will make any sense for the screen reader

File: lib/content-services/src/lib/aspect-list/aspect-list-dialog.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
-<header mat-dialog-title data-automation-id="aspect-list-dialog-title" class="adf-aspect-list-dialog">
-    <h4 class="adf-aspect-list-dialog-title">{{title | translate}}</h4>
+<div mat-dialog-title data-automation-id="aspect-list-dialog-title" class="adf-aspect-list-dialog">
+    <h1 class="adf-aspect-list-dialog-title">{{title | translate}}</h1>
```
Comment (DenysVuika) [inline lib/content-services/src/lib/aspect-list/aspect-list-dialog.component.html:2]: does it still preserve the font size as before?

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: lib/content-services/src/lib/aspect-list/aspect-list-dialog.component.html (hunk 2 above)
Comment (rbahirsheth) [reply lib/content-services/src/lib/aspect-list/aspect-list-dialog.component.html:2]: Yes it looks the same way as before.

File: lib/content-services/src/lib/aspect-list/aspect-list.component.html (hunk 1 above)
Comment (rbahirsheth) [reply lib/content-services/src/lib/aspect-list/aspect-list.component.html:10]: I checked the 'title' works for assistive technology but title property doesn't  supporting to <mat-checkbox>. Please suggest if there is any other solution/approach.

Comment (MichalKinas) [reply lib/content-services/src/lib/aspect-list/aspect-list.component.html:10]: @rbahirsheth I think what Denys is trying to say is that label is correct solution but the label you proposed isn't much human-readable

Comment (rbahirsheth) [reply lib/content-services/src/lib/aspect-list/aspect-list.component.html]: This issue is resolved. Now screen reader is reading checkbox label.

File: lib/content-services/src/lib/aspect-list/services/dialog-aspect-list.service.ts [hunk 3]
```diff
@@ -43,20 +44,23 @@ export class DialogAspectListService {
             nodeId
         };
 
-        this.openDialog(data, 'adf-aspect-list-dialog', '750px');
+        this.openDialog(data, 'adf-aspect-list-dialog', '750px', 'dialog');
         return select;
     }
 
-    private openDialog(data: AspectListDialogComponentData, panelClass: string, width: string) {
+    private openDialog(data: AspectListDialogComponentData, panelClass: string, width: string, role) {
```
[the remark this answers is not in the document]
Comment (rbahirsheth) [reply lib/content-services/src/lib/aspect-list/services/dialog-aspect-list.service.ts]: Updated.

File: lib/content-services/src/lib/aspect-list/services/dialog-aspect-list.service.ts [hunk 4]
```diff
@@ -43,20 +44,23 @@ export class DialogAspectListService {
...
         this.dialog.open(AspectListDialogComponent, {
             data,
             panelClass,
             width,
+            role,
```
[the remark this answers is not in the document]
Comment (rbahirsheth) [reply lib/content-services/src/lib/aspect-list/services/dialog-aspect-list.service.ts]: Updated.

**Review by DenysVuika — APPROVED** [review]

**Review by MichalKinas — APPROVED** [review]

**Review by nikita-web-ua — APPROVED** [review]

**Pull request merged**